main.cpp

```diff
@@ -285,9 +320,13 @@ int main(int argc, char** argv)
     /// Assign Pareto optimal values
     if(prob == 5) {
         cout << "\n% Optimal set cannot be analytically derived for CODeM5" << endl;
+        if (writeJsonFile){
+            jsonOutput["status"] = "Optimal set cannot be analytically derived for CODeM5";
+        }
     }
     else
     {
+        jsonOutput["status"] = "Optimal set can be analytically derived for CODeM5";
```

Comment:
'else' means it is not CODeM5, and therefore the optimal set can be analytically derived.
The 'status' is not correct here